### DISCUSSION TITLE
Add the option to Find and Replace

### DISCUSSION THREAD

**Discussion by rodrcastro** [GitbookIO/community#4] [Feature Requests]
### What's your feature/idea?

Implement a find-and-replace that can be used across the current page.

### What's your desired outcome?

Being able to search for a specific term of phrase that is used multiple times on a page, and easily replace it with something else.

### What's the impact of this for you?

- [ ] Critical
- [ ] High
- [ ] Medium
- [X] Low

### Anything else?

While this is possible if I use Git Sync and a regular text editor, this would be nice to have in the native GitBook editor as well. Not all collaborators edit their content through GitHub, so a native option is always helpful :)

**Comment by CashVo** [comment]
and, add an option to do global find/replace in all pages within a space would be nice. In executing this command, provide a "Preview" of all the changes found and let me decide which instance I don't want to apply the change ... clicking the "Replace" button here would replace all instances that are still "Checked" and clicking "Cancel" will cancel the action. Also, in the preview list, for each page, make it a link so I can jump to the page.
